packages/layout/src/tree-layout/drawing-manager/drawingManager.js

```diff
@@ -0,0 +1,127 @@
+import { HTMLRenderer } from '../renderers/html-renderer';
+import { Utils } from '../utils/utils';
```

Comment:
if the name of the folder is same as file in the folder, use index.js instead of utils.js 
thus, import it from '../utils';